smart_fact_crawler/tests/test_functional.py

```diff
@@ -54,16 +54,38 @@ def test_timestamp_dates():
 
 
 def test_broken_page():
-    sfc.smartfacturl = 'file:' + path.join(
+    test_file_dir = path.join(
         path.dirname(sfc.__file__),
         'resources',
         '20160703_233149_broken_fsc',
         )
 
+    sfc.smartfacturl = 'file:' + test_file_dir
+
     with raises(IndexError):
         sfc.camera_climate()
 
 
+def test_text_in_exception():
+    test_file_dir = path.join(
+        path.dirname(sfc.__file__),
```

Comment:
I have the feeling this should use pkg_resources. I use pkg_resources for this task. But on the other hand dunderscore __file__ dunderscore is more explicit.

Reply:
yes it should. But this PR is not fixing that